ProcessingShowPath/ProcessingShowPath.pde

```diff
@@ -98,13 +98,27 @@ void draw() {
   }
 }
 
+void printParticipantResult(String filename, FloatList wanderTimes)
+{
+  if(backTimeSaved)
+  {
+    println(split(filename, ".")[0] + ", " + forwardTime + ", " + backTime + ", " + mistakes + ", " + shortcuts + ", " + wanderTimes.size() + ", " + wanderTimeSum  + ", " + "Yes");
+  }
+  else 
+  {
+    println(split(filename, ".")[0] + ", " + forwardTime + ", " + backTime + ", " + mistakes + ", " + shortcuts + ", " + wanderTimes.size() + ", " + wanderTimeSum  + ", " + "No");
+  }
+}
```

Comment:
could it just print everything and then based on that `backTimeSaved` it would add "Yes" or "No"? Would make it easier to understand what is different between those lines. Also maybe there's a better way of word usage for that? Not "Yes" and "No", but something else?

Reply:
It is used when analysing the data in the jupyter notebook. We could Have a String variable finished in the processing script and append the generic String. We would end up with same amount of lines, just less repetition i guess. Do you think that would be better?

Reply:
Yeah, I think with less repetition it would be easier to understand what is different between them 👍